### DISCUSSION TITLE
Record a session in 1.2.3

### DISCUSSION THREAD

**Discussion by JohnGooler** [rustdesk/rustdesk#6030] [General]
After update to 1.2.3, "Rec" icon gone and there are no way to record the session.
what should i do to enable Record session?

Thanks

**Comment by 21pages** [comment]
The priority of AV1 is higher than VP9 in auto codec. Currently, AV1 does not support recording, so you can switch to VP9.

**Reply by JohnGooler** [reply]
Wow, fast response.
Thank you very much. it's working now

**Comment by rustdesk** [comment]
@21pages do not disable it, add warning dialog to tell users about this after clicking on. And better way is switching to vp9 automatically for a codec with out recording support after clicking on recording.

**Reply by 21pages** [reply]
https://github.com/rustdesk/rustdesk/pull/6084
